fighthealthinsurance/views.py

```
@@ -631,44 +644,81 @@ def post(self, request):
-                    uuid=session.metadata.get("uuid")
-                ).update(paid=True, should_send=True)
+        try:
+            if domain_id and professional_user_id:
+                user_domain = UserDomain.objects.get(id=domain_id)
+                professional_user = ProfessionalUser.objects.get(
+                    id=professional_user_id
+                )
+                email = professional_user.user.email
             else:
-                logger.error(f"Unknown payment type: {payment_type}")
+                domain_id = resolve_domain_id(
```

Ruff (0.8.2)

Undefined name `resolve_domain_id`

(F821)